lib/context.js

```diff
@@ -1,32 +1,69 @@
 'use strict'
 
-const { kFourOhFourContext, kReplySerializerDefault } = require('./symbols.js')
+const {
+  kFourOhFourContext,
+  kReplySerializerDefault,
+  kSchemaErrorFormatter,
+  kErrorHandler,
+  kReply,
+  kRequest,
+  kBodyLimit,
+  kLogLevel,
+  kContentTypeParser
+} = require('./symbols.js')
 
 // Objects that holds the context of every request
 // Every route holds an instance of this object.
-function Context (schema, handler, Reply, Request, contentTypeParser, config, errorHandler, bodyLimit, logLevel, logSerializers, attachValidation, replySerializer, schemaErrorFormatter, server) {
+function Context ({
+  schema,
+  handler,
+  Reply,
+  Request,
+  contentTypeParser,
+  config,
+  errorHandler,
+  bodyLimit,
+  logLevel,
+  logSerializers,
+  attachValidation,
+  replySerializer,
+  schemaErrorFormatter,
+  server,
+  allowedServerFields
+}) {
+  const getAllowedValue = (passedValue, serverValue, fieldName) => {
+    const isServerFieldNotAllowed = !allowedServerFields.includes(fieldName)
+
+    if (isServerFieldNotAllowed) {
+      return passedValue
+    }
+    return typeof passedValue === 'undefined' ? serverValue : passedValue
+  }
```

Comment:
an you please move this to a top level function?

Reply:
Hi @mcollina I used suggestion from @climba03003, so we don't need this function anymore